engine/src/engine.mlc

```diff
@@ -44,6 +44,8 @@ public foreign handler MCEngineEvalScriptObjectExists(in pObject as ScriptObject
 public foreign handler MCEngineEvalScriptObjectDoesNotExists(in pObject as ScriptObject, out rExists as bool) as undefined binds to "<builtin>"
 public foreign handler MCEngineExecGetPropertyOfScriptObject(in pProperty as string, in pObject as ScriptObject) as any binds to "<builtin>"
 public foreign handler MCEngineExecSetPropertyOfScriptObject(in pProperty as string, in pObject as ScriptObject, in pValue as any) as undefined binds to "<builtin>"
+public foreign handler MCEngineEvalOwnerOfScriptObject(in pObject as ScriptObject, out rParent as ScriptObject) as undefined binds to "<builtin>"
+public foreign handler MCEngineEvalControlsOfScriptObject(in pObject as ScriptObject, out rChildren as list) as undefined binds to "<builtin>"
```

Comment:
This should probably be ChildrenOfScriptObject - if it is the general form discussed (as the children might not be controls!)